lib/web_ui/lib/src/engine/skwasm/skwasm_impl/font_collection.dart

```diff
@@ -3,34 +3,116 @@
 // found in the LICENSE file.
 
 import 'dart:async';
+import 'dart:convert';
+import 'dart:ffi';
+import 'dart:js_interop';
 
 import 'dart:typed_data';
 
 import 'package:ui/src/engine.dart';
+import 'package:ui/src/engine/skwasm/skwasm_impl.dart';
 
 class SkwasmFontCollection implements FlutterFontCollection {
+  SkwasmFontCollection() : _handle = fontCollectionCreate();
+
+  FontCollectionHandle _handle;
```

Comment:
final?

Reply:
`clear` recreates the underlying font collection, so the handle can't be final.